app/dashboards/car_dashboard.rb

```diff
@@ -62,8 +62,11 @@ class CarDashboard < Administrate::BaseDashboard
 
   # Overwrite this method to customize how cars are displayed
   # across all pages of the admin dashboard.
-  #
   # def display_resource(car)
```

Comment:
Remove this type of comments from all Dashboards. It's look strange :) In general we shouldn't leave unnecessary comments. We will talk about it in a Hangout meeting.